groups/groups.yaml

```diff
@@ -595,8 +595,8 @@ groups:
       - spiffxp@gmail.com
       - thockin@google.com
 
-  - email-id: k8s-infra-rbac-wg-reliability@kubernetes.io
-    name: k8s-infra-rbac-wg-reliability
+  - email-id: k8s-infra-rbac-sippy@kubernetes.io
```

Comment:
You'll probably need to rebase, I did this in https://github.com/kubernetes/k8s.io/pull/1844